presto-hive/src/main/java/com/facebook/presto/hive/security/SqlStandardAccessControl.java

```diff
@@ -229,10 +222,9 @@ public void checkCanCreateViewWithSelectFromColumns(ConnectorTransactionHandle t
     }
 
     @Override
-    public void checkCanSetCatalogSessionProperty(Identity identity, String propertyName)
+    public void checkCanSetCatalogSessionProperty(ConnectorTransactionHandle transaction, Identity identity, String propertyName)
```

Comment:
Can `checkCanSetCatalogSessionProperty` be called outside of transaction (so that there is not valid `transaction`)?

Reply:
The caller of this, `AccessControl#checkCanSetCatalogSessionProperty`, has `TransactionId`, so I think there should always be one.